.fixtures.yml

```diff
@@ -2,5 +2,7 @@
 # See https://github.com/puppetlabs/puppetlabs_spec_helper#using-fixtures for details
 ---
 fixtures:
-  forge_modules:
-#     stdlib: "puppetlabs/stdlib"
+  repositories:
+    stdlib: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
+  symlinks:
+    puppet_webhook: "#{source_dir}"
```

Comment:
I think the symlink is created by default these days.

Reply:
https://github.com/puppetlabs/puppetlabs_spec_helper/blob/master/CHANGELOG.md#120---2016-08-23
```
Automatically symlink the module directory, if no symlink fixtures are specified.
```

Reply:
Did not know that, I'll remove it then